edbdeploy/options.py

```diff
@@ -2,8 +2,21 @@
 import re
 import argparse
 import textwrap
+import psutil
```

Comment:
You should add `psutil` to the dependancies listed in the `setup.py` file here: https://github.com/EnterpriseDB/postgres-deployment/blob/master/setup.py#L52